client/my-sites/customer-home/cards/launchpad/intent-write.tsx

```diff
@@ -23,13 +23,14 @@ const LaunchpadIntentWrite = (): JSX.Element => {
 	const completedSteps = ( checklist?.filter( ( task: Task ) => task.completed ) || [] ).length;
 	const tasklistCompleted = numberOfSteps > 0 && completedSteps === numberOfSteps;
 
-	const recordTaskClickTracksEvent = ( task: Task ) => {
+	const recordTaskClickTracksEvent = ( task: Task, order: number ) => {
 		recordTracksEvent( 'calypso_launchpad_task_clicked', {
```

Comment:
I think we should also add the position to the `view` event so we can directly identify which position tasks are in when they are viewed, as they may not be clicked in some positions.